launch/sensor_combined_listener.launch.py

```diff
@@ -52,10 +52,11 @@ def generate_launch_description():
     sensor_combined_listener_node = Node(
         package='px4_ros_com',
         executable='sensor_combined_listener',
-        shell=True
+        output='screen'
+        shell=True,
     )
 
     return LaunchDescription([
-        micro_ros_agent,
+        #micro_ros_agent,
```

Comment:
why not delete?

Reply:
I can delete if that's what it takes to get in. But this is kind of useful to show how you might launch an agent as needed from a ROS script.

If you require it?